cloudferrylib/os/network/neutron.py

```diff
@@ -81,6 +81,9 @@ def read_info(self, **kwargs):
 
         nets = self.get_networks(tenant_id)
         subnets = self.get_subnets(tenant_id)
+        detached_ports = self.get_detached_ports(tenant_id)
+        LOG.debug('List of detached ports: %s',
```

Comment:
Maybe it is better to have this log message inside the `get_detached_ports` method?